src/components/pages/Categories/Categories.vue

```diff
@@ -33,23 +37,39 @@ export default {
     access() {
       return this.$store.getters['auth/access'];
     },
+    category() {
+      const { name } = this.$store.state.categories.category;
+      return name;
```

Comment:
[imo]

以下のようにすれば、1行で済んでシンプルになるかなと思います。

```suggestion
      return this.$store.state.categories.category.name;
```